torch/csrc/tensor/python_tensor.cpp

```diff
@@ -35,6 +35,7 @@ struct PyTensorType {
   THPDtype* dtype;
   THPLayout* layout;
   bool is_cuda;
+  bool is_quantized;
```

Comment:
maybe I missed this in the previous diffs, but do you actually need this?  This is for legacy constructors (e.g. `torch.FloatTensor`) -- if you don't have one of those, you don't need this; python_variable should be enough.

Reply:
I see, I'll remove